packages/controller-utils/src/index.ts

```diff
@@ -1,4 +1,32 @@
 export * from './constants';
-export * from './util';
+export type { NonEmptyArray } from './util';
+export {
+  isSafeDynamicKey,
```

Comment:
Sorry one more request — what do you think about alphabetizing this list?

Reply:
Applied here: https://github.com/MetaMask/core/pull/4041/commits/bad6ed416f82659d9cf6e545f30a6e6d55124c1d